day_two_guide/topics/deprecating_node.adoc

```diff
@@ -0,0 +1,386 @@
+ocp-infra-node-rghb   Ready                      23h       v1.6.1+5115d708d7
+ocp-master-dgf8       Ready,SchedulingDisabled   23h       v1.6.1+5115d708d7
+ocp-master-q1v2       Ready,SchedulingDisabled   23h       v1.6.1+5115d708d7
+ocp-master-vq70       Ready,SchedulingDisabled   23h       v1.6.1+5115d708d7
+ocp-node-020m         Ready                      23h       v1.6.1+5115d708d7
+ocp-node-7t5p         Ready                      23h       v1.6.1+5115d708d7
+ocp-node-n0dd         Ready                      23h       v1.6.1+5115d708d7
+----
++
+As an example, this topic deprecates the `ocp-infra-node-b7pl` infrastructure
+node.
+
+. Describe the node and its running services:
++
+----
+$ oc describe node ocp-infra-node-b7pl
+Name:			ocp-infra-node-b7pl
+Role:
+Labels:			beta.kubernetes.io/arch=amd64
+			beta.kubernetes.io/instance-type=n1-standard-2
+			beta.kubernetes.io/os=linux
+			failure-domain.beta.kubernetes.io/region=europe-west3
+			failure-domain.beta.kubernetes.io/zone=europe-west3-c
+			kubernetes.io/hostname=ocp-infra-node-b7pl
+			role=infra
+Annotations:		volumes.kubernetes.io/controller-managed-attach-detach=true
+Taints:			<none>
+CreationTimestamp:	Wed, 22 Nov 2017 09:36:36 -0500
+Phase:
+Conditions:
+  ...
+Addresses:		10.156.0.11,ocp-infra-node-b7pl
+Capacity:
+ cpu:		2
+ memory:	7494480Ki
+ pods:		20
+Allocatable:
+ cpu:		2
+ memory:	7392080Ki
+ pods:		20
+System Info:
+ Machine ID:			bc95ccf67d047f2ae42c67862c202e44
+ System UUID:			9762CC3D-E23C-AB13-B8C5-FA16F0BCCE4C
+ Boot ID:			ca8bf088-905d-4ec0-beec-8f89f4527ce4
+ Kernel Version:		3.10.0-693.5.2.el7.x86_64
+ OS Image:			Employee SKU
+ Operating System:		linux
+ Architecture:			amd64
+ Container Runtime Version:	docker://1.12.6
+ Kubelet Version:		v1.6.1+5115d708d7
+ Kube-Proxy Version:		v1.6.1+5115d708d7
+ExternalID:			437740049672994824
+Non-terminated Pods:		(2 in total)
+  Namespace			Name				CPU Requests	CPU Limits	Memory Requests	Memory Limits
+  ---------			----				------------	----------	---------------	-------------
+  default			docker-registry-1-5szjs		100m (5%)	0 (0%)		256Mi (3%)0 (0%)
+  default			router-1-vzlzq			100m (5%)	0 (0%)		256Mi (3%)0 (0%)
+Allocated resources:
+  (Total limits may be over 100 percent, i.e., overcommitted.)
+  CPU Requests	CPU Limits	Memory Requests	Memory Limits
+  ------------	----------	---------------	-------------
+  200m (10%)	0 (0%)		512Mi (7%)	0 (0%)
+Events:		<none>
+----
++
+The output above shows that the node is running two pods: `router-1-vzlzq` and
+`docker-registry-1-5szjs`. Two more infrastructure nodes are available to migrate these two pods.
++
+[NOTE]
+====
+The cluster described above is an highly available cluster, this means both the
+`router` and `docker-registry` services are running on all infrastructure nodes.
+====
+
+. Mark a node as unschedulable, and evacuate all of its pods:
++
+----
+$ oadm drain ocp-infra-node-b7pl --delete-local-data
```

Comment:
Do not use oadm - use oc adm. oadm is getting deprecated. You will need to check with eng that a simple substitution of oc adm for oadm will work.

Reply:
@vikram-redhat Thanks. This was in one of the google docs from QE. They suggested it changed to `oc adm`, so that's what I've done.